modules/installation-mirror-repository.adoc

```diff
@@ -74,4 +74,6 @@ $ oc adm release extract --command=openshift-install "${LOCAL_REGISTRY}/${LOCAL_
 To ensure that you use the correct images for the version of {product-title}
 that you selected, you must extract the installation program from the mirrored
 content.
+
+You must perform this step on a machine with an active internet connection.
```

Comment:
Consider combining this with the previous sentence into a single admonition: "...you must extract the installation program from the mirrored content using a machine with an active internet connection."

Reply:
If the major point of the PR wasn't "people are missing that you need to do this on a machine with internet," I'd take that improvement.